components/bms_boss/src/BMS.c

```diff
@@ -20,9 +21,13 @@
 #include "FeatureDefines_generated.h"
 
 #define CURRENT_SENSE_V_per_A 0.005f
+#define PRECHARGE_MIN_TIME_MS 1320U
 
 BMSB_S BMS;
 
+static drv_timer_S precharge_timer;
+static bool precharge_timer_started = false;
+
```

Comment:
You shouldnt need a precharge timer started. You know the timer hasnt started because it should be in a `STOPPED` state